### PR TITLE
Create UserPage US61_SK

### PR DESCRIPTION
**Pull request by latifetor**

### CODE REVIEW

**Comment by TijanaMa** [comment]
looks good

File: src/test/resources/features/US61_UserInfo_SK.feature [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+  US61 As a user, I want to change the number of rows to see the user info
+
+  Background: User is already in the log in page
+    Given the user logged in as "librarian"
+
+
+  Scenario: Change number of rows in User page
+
+    And I click on "User" link
```
Comment (TijanaMa) [inline src/test/resources/features/US61_UserInfo_SK.feature:11]: user

**Review by TijanaMa — APPROVED** [review]

File: src/test/resources/features/US61_UserInfo_SK.feature (hunk 1 above)
Comment (latifetor) [reply src/test/resources/features/US61_UserInfo_SK.feature:11]: According to the userStory "User" being used, I just simply followed

**Review by ekilic76 — APPROVED** [review]

File: src/test/resources/features/US61_UserInfo_SK.feature (hunk 1 above)
Comment (TijanaMa) [reply src/test/resources/features/US61_UserInfo_SK.feature:11]: @latifetor 

    
    You can implement this method in your UserPage ---> 
    
   public  void clickOption(String option){
        String locator="//span[.='"+option+"']";
        BrowserUtils.sleep(1);
        Driver.getDriver().findElement(By.xpath(locator)).click();
    }

    
    And then pass in your  step defs this method , make object from UserPage first---> 
    
      @When("I click on {string} link") -------> this is your step definition
    public void i_click_on_link(String string) {
       usersPage.clickOption(string); 
    }

**Comment by TijanaMa** [comment]
@latifetor implement the method I shared with you here, and I will delete these lines that are making conflict

File: src/test/java/com/phoenix_library/pages/US61_UserPage.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    public void showDropdown(){
+        WebElement showDropdown = Driver.getDriver().findElement(By.name("tbl_users_length"));
+        Select select = new Select(showDropdown);
+        select.selectByIndex(3);
+    }
+
+    @FindBy(xpath = "//option[.='50']")
+    public WebElement recordValue;
+
+
```
Comment (TijanaMa) [inline src/test/java/com/phoenix_library/pages/US61_UserPage.java:23]: public  void clickOption(String option){
        String locator="//span[.='"+option+"']";
        BrowserUtils.sleep(1);
        Driver.getDriver().findElement(By.xpath(locator)).click();
    }

File: src/test/java/com/phoenix_library/step_definitions/US61_UserInfo_StepDefs_SK.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+public class US61_UserInfo_StepDefs_SK {
+
+    US61_UserPage userPage = new US61_UserPage();
+
+
+    @Given("I click on {string} link")
+    public void i_click_on_link(String user) {
+       userPage.userLink.click();
+    }
```
Comment (TijanaMa) [inline src/test/java/com/phoenix_library/step_definitions/US61_UserInfo_StepDefs_SK.java:18]: @Given("I click on {string} link")
    public void i_click_on_link(String user) {
       userPage.clickOption(user);
    }

File: src/test/resources/features/US61_UserInfo_SK.feature (hunk 1 above)
Comment (latifetor) [reply src/test/resources/features/US61_UserInfo_SK.feature:11]: Hi Tijana, can you explain more why do I have to implement as a String option? 
in my US61_UserPage I locate the webElement with I use as "userLink"

**Pull request merged**